service-app/internal/parser/ep2pg_parser/ep2pg_validator_test.go

```diff
@@ -10,44 +9,22 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-var err error
-
 func TestValidXML(t *testing.T) {
 	validator := getValidator(t, "EP2PG-valid.xml")
-	err = validator.Validate()
-	require.NoError(t, err, "Expected no errors")
+	errMessages := validator.Validate()
+	errMessagesLen := len(errMessages)
+	if errMessagesLen > 0 {
+		t.Errorf("Expected no errors but got %d", errMessagesLen)
+	}
```

Comment:
Not essential, but you could collapse this down with `assert.Len`:
```suggestion
	assert.Len(t, validator.Validate(), 0, "Expected no validation errors")
```

Reply:
See https://github.com/ministryofjustice/opg-scanning/pull/81/commits/b950ea3e83603bb0b77aff33c3379dd7c9962a0f